apps/fz_http/priv/repo/migrations/20221207180310_create_gateways.exs

```diff
@@ -0,0 +1,22 @@
+defmodule FzHttp.Repo.Migrations.CreateGateways do
+  use Ecto.Migration
+
+  def change do
+    create table(:gateways, primary_key: false) do
+      add(:id, :uuid, primary_key: true, default: fragment("gen_random_uuid()"))
+      add(:name, :string, null: false)
+      add(:ipv4_masquerade, :boolean, default: true, null: false)
+      add(:ipv6_masquerade, :boolean, default: true, null: false)
+      add(:ipv4_address, :inet)
+      add(:ipv6_address, :inet)
```

Comment:
Should these have defaults as well? You could use the defaults pulled from the corresponding env vars in `config/runtime.exs`